crates/optimism/evm/src/lib.rs

```diff
@@ -30,9 +30,23 @@ pub use error::OptimismBlockExecutionError;
 use revm_primitives::{Bytes, Env, OptimismFields, TxKind};
 
 /// Optimism-related EVM configuration.
-#[derive(Debug, Default, Clone, Copy)]
+#[derive(Debug, Default, Clone)]
```

Comment:
this shouldn't have `Default` too, it causes failing tests

@mattsse I'm wondering if `ChainSpec` should have `Default` at all? feels like `ChainSpec::empty()` would be more explicit and safer

Reply:
@mattsse Should we open an issue about this?